CMakeLists.txt

```diff
@@ -846,7 +846,24 @@ if (ENABLE_PACKAGE_CONFIG)
 			${CMAKE_CURRENT_BINARY_DIR}/SndFileConfigVersion.cmake
 		DESTINATION	${CMAKE_INSTALL_PACKAGEDIR}
 		)
-
+        if (SndFile_WITH_EXTERNAL_LIBS AND NOT BUILD_SHARED_LIBS)
+            install(FILES
+                    ${PROJECT_SOURCE_DIR}/cmake/FindFLAC.cmake
+                    ${PROJECT_SOURCE_DIR}/cmake/FindOgg.cmake
+                    ${PROJECT_SOURCE_DIR}/cmake/FindOpus.cmake
+                    ${PROJECT_SOURCE_DIR}/cmake/FindSndio.cmake
+                    ${PROJECT_SOURCE_DIR}/cmake/FindSpeex.cmake
+                    ${PROJECT_SOURCE_DIR}/cmake/FindVorbis.cmake
+                    DESTINATION ${CMAKE_INSTALL_PACKAGEDIR}
+                    )
+        endif()
+        if (SndFile_WITH_MPEG AND NOT BUILD_SHARED_LIBS)
```

Comment:
@evpobr is there a reason to keep the `AND NOT BUILD_SHARED_LIBS` conditional?

Reply:
I can answer this question too. The reason for this condition is that those cmake modules are only needed when configuring downstream projects in case libsndfile is an installed static library. When libsndfile is an installed shared library the external libraries are private dependencies, and downstream projects will only need the corresponding runtime libraries, but not the headers or the link libraries (which is what the cmake modules are finding).

Reply:
Correct.